src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/pages/Application.vue

```diff
@@ -68,17 +76,51 @@ export default defineComponent({
     const alertStore = useAlertStore();
     const applicationStore = useApplicationStore();
     const loadingStore = useLoadingStore();
+    const certificationStore = useCertificationStore();
 
     // Refresh userProfile from the server
     const userProfile = await getProfile();
     if (userProfile !== null) {
       userStore.setUserProfile(userProfile);
     }
+    const latestCeritificateIsExpired = (dt: string) => {
+      return DateTime.fromISO(dt) > DateTime.fromISO(certificationStore?.latestCertification?.expiryDate!);
+    };
+
+    const latestCeritificateExpiredMoreThan5Years = (dt1: string) => {
+      const dt2 = DateTime.fromISO(certificationStore.latestCertification?.expiryDate!);
+      const differenceInYears = Math.abs(DateTime.fromISO(dt1).diff(dt2, "years").years);
+      return differenceInYears > 5;
+    };
 
```

Comment:
to be discussed, I think we need to also handle loading the appropriate config file for the Wizard component at the top of the file. 

Also the initializeWizard might need it's own method for renewal applications that are different.

Reply:
Let's handle this as we work through the renewal flows